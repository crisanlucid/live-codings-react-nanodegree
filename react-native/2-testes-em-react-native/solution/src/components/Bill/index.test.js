import React from 'react';
import { create } from 'react-test-renderer';
import Bill from './';

describe('[Component] Bill', () => {
  const bill = {
    dueDay: '11',
    label: 'Rent',
  };

  it('matches snapshot', () => {
    const tree = create(<Bill bill={bill} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
