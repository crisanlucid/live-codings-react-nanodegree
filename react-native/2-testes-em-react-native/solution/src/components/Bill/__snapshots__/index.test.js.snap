// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Component] Bill matches snapshot 1`] = `
<View
  style={
    Object {
      "display": "flex",
      "padding": 4,
      "width": "100%",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "space-between",
        "width": "100%",
      }
    }
  >
    <Text
      style={
        Object {
          "fontSize": 24,
        }
      }
    >
      Rent
    </Text>
    <Text>
      Due on  11
    </Text>
  </View>
</View>
`;
